Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransferInfo.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Frends.SFTP.DownloadFiles.Definitions
+{
+    internal class FileTransferInfo
+    {
+        public string TransferName { get; set; } // 0
```

Comment:
At this point I do not know what the //0 and //1 and //2 mean in this file and I am too afraid to ask :)

Reply:
I have no idea. I copied this from Cobalt. Will delete them.